
target1_x : 0.0
target1_y : 1.65
target2_x : 1.85
target2_y : 1.65
target3_x : 1.85
target3_y : -1.6
target4_x : 0.0
target4_y : -1.6

square_size: 0.4
err_max: 0.1
fly_height: 0.55
